app/helpers/application_helper.rb

```diff
@@ -149,4 +149,8 @@ def default_meta_description
   def present(model)
     yield(model.presenter)
   end
+
+  def user_can_edit?
```

Comment:
do we need this both in application helper and in static pages helper?

Reply:
Would this be best to leave in application_helper so we can use it everywhere?